p2p/src/lib.rs

```diff
@@ -203,32 +203,13 @@ where
                     consensus_handle,
```

Comment:
We're still calling this consensus for some reason... please rename it to chainstate_handle